src/encoded/tests/test_upgrade_characterization.py

```diff
@@ -101,6 +101,24 @@ def antibody_characterization_10(antibody_characterization_1):
     return item
 
 
+@pytest.fixture
+def antibody_characterization_11(antibody_characterization):
+    item = antibody_characterization.copy()
+    item.update({
```

Comment:
pep8 issues on indents and brackets...etc
```
    item.update({
        'characterization_reviews': [
            {
                'biosample_term_name': 'K562',
                'biosample_term_id': 'EFO:0002067',
                'lane_status': 'exempt from standards',
                'biosample_type': 'immortalized cell line',
                'lane': 2,
                'organism': '/organisms/human/'
            }
        ]
    })
```

Reply:
I have corrected & pushed the fix